### ISSUE TITLE
PLCC value do not meet expectations

### ISSUE THREAD

**Issue by hello02world** [junyongyou/lsct_phiqnet#2]
I read your paper, and trained the model of my own. I adopted the pretrain model PHIQNet and hyper-parameters of Conv1D{filters=[32,64]} and LSCT{L=2, D=64, heads=4, dff=64} writed in your code, and then I tested for 10 random train-test splits, but the average evaluation result PLCC is about 0.83. In some splits, the PLCC values are around ~0.80, but no PLCC value greater than 0.87. Then I tuned different hyper-parameters, still can't reach the average level of 0.86. Is it because of the random seed or the pre-trained model is not good enough? Or other affections?
